### PR TITLE
Fisteele vm vmss orchestrator

### PR DESCRIPTION
**Pull request by fitzgeraldsteele**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [X ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 201-vm-vmss-orchestrator/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+              }
+            }
+          }
+        ],
+        "dnsSettings": {
+          "dnsServers": []
+        },
+        "enableIPForwarding": false
+      },
+      "dependsOn": [
```
Comment (bmoore-msft) [inline 201-vm-vmss-orchestrator/azuredeploy.json]: move this above properties

File: 201-vm-vmss-orchestrator/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+        },
+        "diagnosticsProfile": {
+          "bootDiagnostics": {
+            "enabled": true,
+            "storageUri": "[reference(resourceId('Microsoft.Storage/storageAccounts', variables('diagnosticStorageAccountName')), '2016-01-01').primaryEndpoints.blob]"
+          }
+        }
+      },
+      "dependsOn": [
+        "[concat('Microsoft.Compute/virtualMachineScaleSets/', parameters('virtualMachineScaleSetName'))]",
```
Comment (bmoore-msft) [inline 201-vm-vmss-orchestrator/azuredeploy.json]: remove concat and use resourceId for the dependencies

File: 201-vm-vmss-orchestrator/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```
Comment (bmoore-msft) [inline 201-vm-vmss-orchestrator/azuredeploy.parameters.json]: remove the parameters for which you have defaultValues

File: 201-vm-vmss-orchestrator/azuredeploy.parameters.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "virtualMachineAdminUserName": {
+      "value": "azadmin"
```
Comment (bmoore-msft) [inline 201-vm-vmss-orchestrator/azuredeploy.parameters.json]: use GEN-UNIQUE

File: 201-vm-vmss-orchestrator/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,304 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineAdminUserName": {
```
[the remark this answers is not in the document]
Comment (fitzgeraldsteele) [reply 201-vm-vmss-orchestrator/azuredeploy.json]: Done

File: 201-vm-vmss-orchestrator/README.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# Add multiple VMs into a Virtual Machine Scale Set
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-vm-copy-vmss/PublicLastTestDate.svg" />&nbsp;
```
[the remark this answers is not in the document]
Comment (fitzgeraldsteele) [reply 201-vm-vmss-orchestrator/README.md]: fixed

File: 201-vm-vmss-orchestrator/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,304 @@
...
+        "description": "Virtual Machine Size"
+      }
+    },
+    "operatingSystem": {
+      "type": "string",
+      "defaultValue": "Server2016",
+      "metadata": {
+        "description": "Operating System of the Server"
+      },
+      "allowedValues": [
```
[the remark this answers is not in the document]
Comment (fitzgeraldsteele) [reply 201-vm-vmss-orchestrator/azuredeploy.json]: I wanted to use the same values that are in CLI. But fixed the defaultValue

File: 201-vm-vmss-orchestrator/azuredeploy.json (hunk 2 above)
Comment (fitzgeraldsteele) [reply 201-vm-vmss-orchestrator/azuredeploy.json]: Done

File: 201-vm-vmss-orchestrator/azuredeploy.json (hunk 1 above)
Comment (fitzgeraldsteele) [reply 201-vm-vmss-orchestrator/azuredeploy.json]: Done

**Comment by fitzgeraldsteele** [comment]
OK, got a clean build...

**Pull request merged**